pytorch_lightning/core/mixins/hparams_mixin.py

```diff
@@ -28,7 +28,7 @@ class HyperparametersMixin:
 
     def __init__(self) -> None:
         super().__init__()
-        self._log_hyperparams = True
+        self._log_hyperparams = False
```

Comment:
Can you remind us where do we check whether we need to log or not? I.e., given this fix, there must be a check somewhere whether params were logged or not.

Reply:
here: https://github.com/PyTorchLightning/pytorch-lightning/blob/46d6fbf11b7c1d04edefa121675751db8ca02b18/pytorch_lightning/trainer/trainer.py#L1193-L1203

plus I have updated a test for it.